app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -71,7 +71,7 @@
             android:layout_marginTop="32dp"
             android:contentDescription="@string/required_fields_content_description"
             android:text="@string/add_profile_required"
-            android:textColor="@color/oppia_primary_text"
+            android:textColor="@color/oppia_primary_text_color"
```

Comment:
'primary_text_color' seems too generic here and is more like a color_palette color than a color component one. A color component might be more like 'label_text_color'. In many cases, the style might give a clue on how to name the color.

Reply:
Done.